.github/dependabot.yml

```diff
@@ -1,26 +1,40 @@
 # Please see the documentation for all configuration options:
-# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
 
 version: 2
 updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      # Check for updates to GitHub Actions every week
       interval: "weekly"
+
   - package-ecosystem: "pip"
     directory: "/"
     schedule:
-      interval: "weekly"
+      interval: "monthly"
+    reviewers:
+      - ben-alkov
+      - brunoapimentel
+      - ejegrova
+      - eskultety
+      - slimreaper35
+      - taylormadore
     groups:
-      dep-patch-updates:
-        update-types:
-          - "patch"
-      dep-minor-updates:
+      minor-and-patch:
+        patterns:
+          - "*"
```

Comment:
Isn't `patterns=*` the default (although the docs doesn't say that explicitly)? Also, do we need to specify this field explicitly? The way I understand
> You must use the patterns, exclude-patterns, dependency-type, or update-types options to define the group, or any combination thereof.

is that `update-types` is enough for us to successfully define a group in dependabot config.